### DISCUSSION TITLE
Menubar updates & various fixes - v1.0.0-alpha.3

### DISCUSSION THREAD

**Discussion by AdelaideSky** [Adelenade/GoXlr-Macos#19] [General]
Added On-screen faders, quick routing, launch on device connects (and settings tab as well), UI lock when device not connected...
Also fixed goxlr-mini major bugs (still buggy tho).

Just run the installer, and everything should install as usual !

<hr /><em>This discussion was created from the release <a href='https://github.com/Adelenade/GoXlr-Macos/releases/tag/v1.0.0-alpha.3'>Menubar updates & various fixes - v1.0.0-alpha.3</a>.</em>

**Comment by leanderkretschmer** [comment]
why only macos 13

**Reply by AdelaideSky** [reply]
Because I use in the code a lot of macOS-13-only functionalities of SwiftUI, for example I can't make the same menu bar based app for macOS 13 or have the same design !

**Comment by Almonte007** [comment]
I can't get it working
